test-data/genops-statements.test

```diff
@@ -102,19 +104,21 @@ L0:
 L1:
     r1 = 5
     r2 = n < r1 :: int
-    if r2 goto L2 else goto L6 :: bool
+    if r2 goto L2 else goto L8 :: bool
 L2:
 L3:
     r3 = 4
     r4 = n < r3 :: int
-    if r4 goto L4 else goto L5 :: bool
+    if r4 goto L4 else goto L6 :: bool
 L4:
-    goto L5
-    goto L3
-L5:
     goto L6
-    goto L1
+L5:
+    goto L3
 L6:
+    goto L8
+L7:
+    goto L1
+L8:
```

Comment:
These dead code blocks are ugly, but so were the double gotos and some of our existing empty blocks, and it doesn't really matter at all. We can do something to clean them up later, if we care.